intranet/apps/auth/views.py

```diff
@@ -183,6 +185,21 @@ def post(self, request):
 
             default_next_page = "index"
 
+            if request.user.is_student:
+                if settings.ENABLE_PRE_EIGHTH_REDIRECT:
```

Comment:
Is there a need for an extra layer of indentation? You could just make it `if  request.user is_student and settings.ENABLE_PRE_EIGHTH_REDIRECT:`